themes/default/data/registry/packages/aws-static-website.yaml

```diff
@@ -0,0 +1,14 @@
+category: Cloud
+component: true
+description: ""
+featured: false
+logo_url: ""
+name: aws-static-website
+native: true
```

Comment:
This is not a native provider so the conponent is missing a few keywords - see aws-apigateway schema for details

Reply:
true... thank you.